doc/rst/meta/static/portability-viz.js

```diff
@@ -0,0 +1,356 @@
+(() => {
+  const treeData = {
+    name: "Chapel",
+    children: [
+      {
+        name: "Desktop/Laptop",
+        children: [
+          {
+            name: "Linux",
+            classification: ["GPU"],
+          },
+          {
+            name: "Windows (with WSL)",
+            classification: ["GPU"],
+          },
+          {
+            name: "MacOS",
+          },
+        ],
+      },
+      {
+        name: "Cloud",
+        children: [
+          {
+            name: "AWS",
+            classification: ["GPU", "EFA", "Ethernet"],
+          },
+          {
+            name: "Azure",
+            classification: ["Ethernet"],
+          },
+        ],
+      },
+      {
+        name: "Supercomputer",
+        children: [
+          {
+            name: "Cray XC",
+            classification: ["GPU", "Aries"],
+          },
+          {
+            name: "HPE Apollo",
+            classification: ["GPU", "InfiniBand", "Slingshot"],
+          },
+          {
+            name: "HPE Superdome Flex",
+          },
+          {
+            name: "HPE Cray EX",
+            classification: ["GPU", "Slingshot"],
+          },
+          {
+            name: "Cray CS",
+            classification: ["GPU", "InfiniBand"],
+          },
+        ],
+      },
+      {
+        name: "Cluster",
+        classification: ["GPU", "InfiniBand", "Ethernet"],
+      },
+      {
+        name: "Container",
+        children: [
+          {
+            name: "Docker",
+          },
+        ],
+      },
+    ],
+  };
```

Comment:
Comments on the classification:
* Do we not support GPUs on Macs?  :(
* I'm fairly certain HPE Apollo does not support Slingshot, only IB (and maybe Ethernet?); reviewing HPE's public pages seems to corroborate this.

Reply:
> Do we not support GPUs on Macs? :(
We do not support Apple Silicon gpus, except with CHPL_GPU=cpu :(. We may be able to work with Intel Macs with GPUs, but I don't know if we have ever tested this.

> I'm fairly certain HPE Apollo does not support Slingshot, only IB (and maybe Ethernet?); reviewing HPE's public pages seems to corroborate this.
I will fix this